src/gpl/src/timingBase.cpp

```diff
@@ -186,7 +186,12 @@ TimingBase::updateGNetWeights(float overflow) {
         gNet->setTimingWeight(weight);
         weighted_net_count++;
       }
-      debugPrint(log_, GPL, "replace", 3, "timing: net:{} slack:{} weight:{}", 
+
+      int level = 3;
+      if (sta::fuzzyEqual(gNet->totalWeight(), 1.0))
+        level = 4;
+
+      debugPrint(log_, GPL, "replace", level, "timing: net:{} slack:{} weight:{}",
```

Comment:
What is the level conditional?

Reply:
The idea was to keep the previous behaviour of printing every net but at a higher debug level. I'm not sure there is a lot of point though.

Reply:
Its just a debug so let's pick one level.

Reply:
I've removed this part of the patch.